docker/files/jazzy_base.repos

```diff
@@ -0,0 +1,17 @@
+repositories:
+  beluga:
+    type: git
+    url: https://github.com/Ekumen-OS/beluga.git
+    version: b6d5ee387cd196aa3c11617c993aa6300c7a9484
+  apriltag_ros:
+    type: git
+    url: https://github.com/Adlink-ROS/apriltag_ros.git
+    version: foxy-devel
+  gz_ekumen_worlds:
+    type: git
+    url: https://github.com/Ekumen-OS/gz_ekumen_worlds.git
+    version: main
```

Comment:
Hash?

Reply:
Done in https://github.com/Ekumen-OS/beluga-demos/pull/11/commits/c932c6d8bbff922bb3c1bff16fed9476c603051e.